cmd/run.go

```
@@ -274,11 +281,34 @@ func pollTask(c *client.Client, id string, timeout time.Duration, showlogs, remo
 
 		fmt.Println("ID: ", item.ID, " Update at: ", item.UpdatedAt)
 		for {
+			if len(taskRequest.HeartbeatUrl) > 0 && len(taskRequest.JobToken) > 0 {
+				heartbeatClient := &http.Client{}
+				heartbeatRequest, hrError := http.NewRequest(http.MethodPut, taskRequest.HeartbeatUrl, bytes.NewBufferString("{}"))
+				if hrError != nil {
+					log.Fatal(hrError)
+				}
+				heartbeatRequest.Header.Set("Content-Type", "application/json; charset=utf-8")
+				heartbeatRequest.Header.Set("Authorization", "Bearer "+taskRequest.JobToken)
+				_, _ = heartbeatClient.Do(heartbeatRequest)
```

you ignore an error here, bad idea :) Please log it at least.